src/maintainer/updating_pkgs.rst

```diff
@@ -187,6 +187,17 @@ Note that for long build logs one can do
 
 to save it in a text file for future inspection.
 
+Downloading prebuilt packages
+----------------
+
+To download prebuilt packages follow the steps below:
+
+- Starting from your PR, navigate to the CI.
```

Comment:
you see, you are talking about a PR here but there was no mention earlier that those pre-built packages come from your failled PR attempt, at least in my perspective :smile:. What do you think?